### ISSUE TITLE
Use a different versioning format

### ISSUE THREAD

**Issue by Jan200101** [R2Northstar/NorthstarLauncher#723]
Northstar doesn't really make use of any real versioning format with the version just being incremented as is felt.

I am in favort of adopting some versioning schema already applied to the real work like:
- semver
- calver
- WendtVer
- GitVer 
- pragmaticversioning 
- even/odd versioning ([SDL](https://wiki.libsdl.org/SDL2/README/versions))
- ZeroVer

**Comment by GeckoEidechse** [comment]
I guess that would apply to all of Northstar and not just launcher, right? ^^

**Comment by Jan200101** [comment]
correct, would probably be better in the Northstar repo so I'm gonna transfer it
